python/Ganga/test/GPI/Monitoring/TestMonitoring.py

```diff
@@ -89,7 +90,7 @@ def test_f_reallyDisabled(self):
         j=Job()
         j.submit()
 
-        time.sleep(20.)
+        dummySleep(j, states=['submitted'])
```

Comment:
Surely this line shouldn't be needed if we're going straight from submit to check the status?
I think this was an issue with something not updating live in the past but we can probably drop this now (for a saving of probably 0.1s)